README.md

```diff
@@ -1,4 +1,6 @@
 <h1>COD-FE</h1>
+<img src="https://github.com/atlp-rwanda/cod-be/actions/workflows/node.js.yml/badge.svg">
+<img src="https://img.shields.io/endpoint?url=https://gist.githubusercontent.com/NNesta/bb0f2667445560c93664f6fc01a3a352/raw/cod-fe__pull_3.json">
```

Comment:
This coverage will always be for pull request 3?

Reply:
I corrected it